Sources/xcresultparser/CoberturaCoverageConverter.swift

```diff
@@ -60,44 +111,55 @@ public class CoberturaCoverageConverter: CoverageConverter, XmlSerializable {
         let packagesElement = XMLElement(name: "packages")
         rootElement.addChild(packagesElement)
 
+        // Get the xccov results as a JSON.
+        var arguments = ["xccov", "view"]
+        if resultFile.url.pathExtension == "xcresult" {
+            arguments.append("--archive")
+        }
+        arguments.append("--json")
+        arguments.append(resultFile.url.path)
+        let coverageData = try Shell.execute(program: "/usr/bin/xcrun", with: arguments)
+        let resultsString = String(decoding: coverageData, as: UTF8.self)
```

Comment:
No need to convert coverageData to String and then back to Data.
Instead we can use 
```
guard let coverageJson = try? JSONDecoder().decode(FileCoverage.self, from: coverageData) else
...
```
right away. Right?
Skip line 122 to 127